config.go

```diff
@@ -0,0 +1,137 @@
+package main
+
+import (
+	"github.com/jawher/mow.cli"
+)
+
+type serviceConfig struct {
+	appSystemCode            *string
```

Comment:
every member of the struct is a pointer because if it is not, the cli logic for reading the app configuration will not work. If I dereference the pointers at the end of `createServiceConfiguration`, they will all have empty values because the application startup has not yet taken place.